modules/core/src/test/java/org/apache/ignite/internal/metric/CacheMetricsAddRemoveTest.java

```diff
@@ -136,6 +136,44 @@ public void testCacheMetricsConfigurationNotRemovedOnStop() throws Exception {
         checkMetricsEmpty(cachePrefix);
     }
 
+    /** */
+    @Test
+    public void testCacheMetricsConfigurationNotRemovedOnStopTwoCachesPerGroup() throws Exception {
+        String cachePrefix = cacheMetricsRegistryName("other-cache", false);
```

Comment:
Should we use nearEnabled flag instead of `false` here?

Reply:
Then we should fix `checkMetrics*` methods and other tests to provide the correct cache prefix. I think this is not related to the issue and should be done in a separate issue.